netCDF4/_netCDF4.pyx

```diff
@@ -2202,7 +2202,14 @@ strings.
             elif diskless:
                 ierr = nc_open(path, NC_NOWRITE | NC_DISKLESS, &grpid)
             else:
-                ierr = nc_open(path, NC_NOWRITE, &grpid)
+                if mode == 'rs':
+                    # NC_SHARE is very important for speed reading
+                    # large netcdf3 files with a record dimension.
+                    # Opening as r+s or as implies capability to
+                    # which may be inconsistent with actual access
```

Comment:
Not sure what this means - 'capability to which may be inconsistent with actual access'?
Would be useful to have a reference to this pull request in the comments.

Reply:
@jswhit - I have just returned from vacation and looking at your comment. I haven't done anything with the changelog, but am happy to.

What I meant was that the append modes (a or r+) imply that the user has the capability to write. So, any properties tied to the open mode will expect write capability. If the user doesn't have permission, there is an inconsistency there.

Sorry for the delay.

Reply:
@jswhit - Since you've merged it, do you still want me to fill out a log?

Reply:
No worries, I did it.